app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -38,6 +38,7 @@ class ManageIQ::Providers::Kubevirt::InfraManager < ManageIQ::Providers::InfraMa
           :authentication_token,
           :authentications,
           :endpoints,
+          :default_endpoint,
```

Comment:
This was required for the FactoryBot definition to not blow up when setting `hostname=` in the specs